internal/pips/pips.go

```diff
@@ -0,0 +1,61 @@
+package pips
```

Comment:
I often find it difficult to name an instance of `sip.SIP` because the natural `sip := sip.SIP` stomps the package name. :(

I would much rather change `sip` -> `sips` for a package name rather than renaming `pips` -> `pip`.

Reply:
At some point that may happen with a slice of PIPs, not here, at least for now, just a thought. Not a big deal in any case, up to you.